sdks/java/io/hbase/src/main/java/org/apache/beam/sdk/io/hbase/HBaseIO.java

```diff
@@ -107,6 +107,16 @@
  *         .withFilter(filter));
  * }</pre>
  *
+ * <p>{@link HBaseIO#readAll()} allows to execute multiple {@link Scan}s to multiple {@link Table}s.
```

Comment:
Please add a note that this only works in runners supporting SDF, and link to the compatibility matrix.

Reply:
done